src/libraries/System.Text.Json/tests/Serialization/CustomConverterTests.Int32.cs

```diff
@@ -11,30 +11,45 @@ public static partial class CustomConverterTests
         /// <summary>
         /// Allow both string and number values on deserialize.
         /// </summary>
-        private class Int32Converter : JsonConverter<int>
+        private class Int32Converter : JsonConverterFactory
```

Comment:
This test should be able to derive from `JsonConverter<int>` instead of the factory. Then no `CanConvert` or `CreateConverter` is necessary.

Reply:
Fixed.